src/main/java/org/wso2/carbon/module/swiftiso20022/constants/ConnectorConstants.java

```diff
@@ -28,6 +28,99 @@ private ConnectorConstants() {
+    public static final String MT940_FORWARD_AVAIL_BAL = ":65:";
+    public static final String COLON = ":";
+    public static final String SLASH = "/";
+    public static final String DOUBLE_SLASH = "//";
+    public static final String TRANSACTION_REF = "Transaction Reference Number";
+    public static final String RELATED_REF = "Related Reference";
+    public static final String ACC_IDENTIFICATION = "Account Identification";
+    public static final String STATEMENT_NUMBER = "Statement Number";
+    public static final String OPENING_BALANCE = "Opening Balance";
+    public static final String AMOUNT = " Amount";
+    public static final String CLOSING_BALANCE = "Closing Balance";
+    public static final String CLOSING_AVAIL_BALANCE = "Closing Available Balance";
+    public static final String FORWARD_CLOSING_AVAIL_BALANCE = "Forward Available Balance";
+    public static final String STATEMENT_LINE = "Statement Line";
+    public static final String STATEMENT_NO = "statementNumber";
+    public static final String OPENING_BAL = "openingBalance";
+    public static final String CLOSING_BAL = "closingBalance";
+    public static final String CLOSING_AVAIL_BAL = "closingAvailableBalance";
+    public static final String FORWARD_AVAIL_BAL = "forwardAvailableBalance";
+    public static final String STATEMENT_LINES = "statementLines";
+    public static final String CURRENT_STATEMENT_TYPE = "current";
+    public static final String LAST_STATEMENT_TYPE = "last";
+
+    //Common Error Constants
     public static final String ERROR_CODE = "ERROR_CODE";
     public static final String ERROR_MESSAGE = "ERROR_MESSAGE";
+    public static final String MISSING_REQUEST_PAYLOAD = "missing_request_payload";
+    public static final String INVALID_REQUEST_PAYLOAD = "invalid_request_payload";
+    public static final String SERVER_ERROR = "server_error";
+    public static final String ERROR_VALIDATION_FAILED = "Validation failed";
+    public static final String ERROR_MISSING_PAYLOAD = "Missing Request Payload";
+    public static final String PROCESSING_ERROR = "Error while processing the request";
+    public static final String ERROR_PARAMETER_MISSING = "Missing mandatory parameter %s";
+    public static final String ERROR_PARAMETER_LENGTH = "%s should not contains more than %s characters";
+    public static final String ERROR_PARAMETER_CONTAINS_SLASH = "%s should not contain slashes";
+    public static final String ERROR_PARAMETER_INVALID = "Field %s is invalid";
+    public static final String ERROR_CURRENCY_CODE_INVALID = "Currency code in the field %s is not in ISO 4217" +
+            " format";
+    public static final String ERROR_AMOUNT_NULL = "Amount cannot be null";
+    public static final String ERROR_AMOUNT_SIZE_INVALID = "Amount should not be empty";
+
+    // MT940 Error Constants
+    public static final String ERROR_C24 = "C24";
+    public static final String ERROR_C277 = "C27";
```

Comment:
is `ERROR_C277` correct?

Reply:
Fixed in https://github.com/wso2-extensions/mediation-swift-iso20022-module/pull/2/commits/8d6b87e8988955fbe9679e7afa9d4285b0ff0ac8